src/urllib3/util/connection.py

```diff
@@ -1,11 +1,15 @@
 import socket
+from typing import List, Optional, Tuple
 
 from urllib3.exceptions import LocationParseError
 
 from .wait import wait_for_read
 
+SOCKET_GLOBAL_DEFAULT_TIMEOUT = socket._GLOBAL_DEFAULT_TIMEOUT  # type: ignore
```

Comment:
mypy returns an error - it cannot find socket._GLOBAL_DEFAULT_TIMEOUT, hence `type: ignore`. Should urllib3's own GLOBAL_DEFAULT_TIMEOUT be defined?

Reply:
Nah, using a type ignore for this case is fine.